docker/ct-woodpecker/Dockerfile

```diff
@@ -14,9 +14,11 @@ RUN go install -v ./cmd/ct-woodpecker/...
 ## main
 FROM alpine:3.8
 
-RUN apk update && apk add --no-cache --virtual ca-certificates
+RUN apk update && apk add ca-certificates bash
```

Comment:
What is the rationale for removing `--no-cache` and `--virtual`?  I think I cargo-cult'd this from somewhere else but if you know why it isn't correct I'd be interested in learning.

Reply:
Honestly I cargo-culted this too. My builds weren't working and I didn't recognize these so I pulled them out. `--no-cache` seemed weird since a clean container build shouldn't have cache lying around. I wasn't able to figure out what `--virtual` does. It seems like it maybe allows easily deleting a package and its dependencies, which we don't appear to need.

Reply:
I'm convinced deleting these args is the best idea :+1: 

I don't see `--virtual` in any of the `ct-woodpecker`/`pebble` `Dockerfile`'s, just `--no-cache`. Chasing that cleanup is probably more trouble than its worth.